### PR TITLE
Teach orka redis

### PR DESCRIPTION
**Pull request by nikostoulas**
Redis initialization through orka similar to how mongodb works.
Because redis initialization is more difficult we export 2 methods getRedis to get the client created by orka and createRedisConnection to create extra connections.

To enable ssl you need to set 3 env variables:

```
REDIS_OPTIONS_TLS_CA
REDIS_OPTIONS_TLS_CERT
REDIS_OPTIONS_TLS_KEY
```

### CODE REVIEW

File: src/initializers/redis.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+function getRedisUrl(config) {
+  return config && config.url;
+}
+
+function getHost(url) {
+  return url.split('@')[1] || url;
+}
+
+const exhastError = new Error('Retry retry_strategy options.total_retry_time exhausted');
```
Comment (alkiskal) [inline src/initializers/redis.ts]: `exhaust`?

File: README.md [hunk 2]
```diff
@@ -29,6 +29,14 @@ config = {
   app: {
     name: 'orka', // will be used as newrelic name, rabbit-queue prefix
     env: 'demo' // will be used as newrelic, honeybadger env - defaults to NODE_ENV
+  },
```
Comment (klesgidis) [inline README.md]: 💯

File: src/initializers/redis.ts (hunk 1 above)
Comment (nikostoulas) [reply src/initializers/redis.ts]: What do you mean @alkiskal ?

Comment (klesgidis) [reply src/initializers/redis.ts]: typo

Comment (nikostoulas) [reply src/initializers/redis.ts]: Thanks fixed

**Pull request merged**